docs/architecture/decisions/0002-use-core-and-security-approved-contributed-modules-when-possible.md

```diff
@@ -0,0 +1,27 @@
+# 1. Use core and security-approved contributed modules when possible
+
+## Status
+
+Proposed
+
+## Context
+HSDP has a mix of custom modules, core modules, and contributed modules. 
+
+Custom solutions have offered us the flexibility to tailor functionality to specific needs and requests; however, they can introduce potential challenges, including increased maintenance complexity and associated security considerations.
+
+Core and security-approved contributed modules and related plugins, allow us to leverage community support and regular updates to enhance maintainability and project stability.
+
+## Decision
+We have decided to use core Drupal modules and security-approved contributed modules whenever possible, opting for custom development only when no suitable core or contributed options are available. This approach is intended to ensure better maintainability, minimize risks associated with custom code, and align with best practices within the Drupal community.
+
```

Comment:
This might be a good place to either define the criteria for what contrib modules we'd include or we could follow up with a second ADR to define that criteria. Criteria would include things like:

* Number of sites the module is used on
* The module has to have opted into Security Coverage
* The module must have a stable version (would we consider using alpha or beta modules?)
* The frequency of releases or when the last module was released
* Module level of complexity and any concerns about conflicts with existing modules or configuration
* The number of unresolved bugs and tickets and the criticality of those tickets (avoid module with Major or Critical bug tickets in the current release).

Reply:
How should we phrase this?  As 6 different thresholds to meet?  Or as "these are the things we consider as a total package"?  The latter would give us the flexibility to make a judgment call based on circumstances (e.g. "I know there's a major issue in the queue, but it's to write better documentation").

Reply:
Maybe add something about closely it fits our needs.  If something is almost but not quite what we need, but meets all these maintenance requirements, should we use it? 

What if a contrib module doesn't meet our criteria, but there's no alternative?  Using it is might be less effort in the long run than doing something completely custom.

Reply:
I agree and I don't think any of the individual criteria as binding. They should be taken as a whole with context to make an informed decision based on the pros and cons.